Feat: allow containers to ignore cron (#5)
fbrettnich/hcloud-snapshot-as-backup: README.md

Replace `YOUR-API-TOKEN` with your [Hetzner Cloud Console API-Key](#generate-hetzner-cloud-console-api-key).

> Optional: use `--env CRON=false` to schedule outside of the container, especially using services like [Google Cloud Run jobs](https://cloud.google.com/run/docs/create-jobs) or [Amazon ECS scheduled tasks](https://docs.aws.amazon.com/AmazonECS/latest/userguide/scheduled_tasks.html)

Reviewer: ```suggestion
Optional: Set `CRON` to `false` to disable CronScheduler in the container and schedule outside of the container, especially using services like [Google Cloud Run jobs](https://cloud.google.com/run/docs/create-jobs) or [Amazon ECS scheduled tasks](https://docs.aws.amazon.com/AmazonECS/latest/userguide/scheduled_tasks.html)
```

| %timestamp% | Current timestamp  |

### Docker: Specify how often the script should be executed via cron

> If you run the script in Docker, a cron may be specified in the environment variables under `CRON`.  
> The cron is used to define how often and at which times the script should be executed.  
> Here you can create a cron: [Cron Generator](https://crontab.guru/#0_1_*_*_*)  

Reviewer: ```suggestion
If you run the script in Docker, a cron may be specified in the environment variables under `CRON`.  
The cron is used to define how often and at which times the script should be executed.  
Optionally, set `CRON` to `false` to disable CronScheduler in the container and schedule outside of the container.  

Here you can create a cron: [Cron Generator](https://crontab.guru/#0_1_*_*_*)  
```

For example:  
```
0 1 * * *
```

> This cron executes the script every day at 1am. Optionally, set `CRON` to `false` to disable CronScheduler in the container and schedule outside of the container

Reviewer: ```suggestion
This cron executes the script every day at 1am.  
```
